plugins/file/plugin.go

```diff
@@ -125,24 +162,41 @@ func setupFile6(args ...string) (handler.Handler6, error) {
 }
 
 func setupFile4(args ...string) (handler.Handler4, error) {
-	log.Print("plugins/file: loading `file` plugin for DHCPv4")
-	return nil, nil
+	_, h4, err := setupFile(false, args...)
+	return h4, err
 }
 
 func setupFile(v6 bool, args ...string) (handler.Handler6, handler.Handler4, error) {
-	if len(args) < 1 {
-		return nil, nil, errors.New("plugins/file: need a file name")
-	}
-	filename := args[0]
-	if filename == "" {
-		return nil, nil, errors.New("plugins/file: got empty file name")
-	}
-	records, err := LoadDHCPv6Records(filename)
-	if err != nil {
-		return nil, nil, fmt.Errorf("plugins/file: failed to load DHCPv6 records: %v", err)
+	if v6 {
```

Comment:
This can be simplified since v6 and v4 share some logic:

```
if len(args) < 1 {
        return nil, nil, errors.New("plugins/file: need a file name")
}
filename := args[0]
if filename == "" {
        return nil, nil, errors.New("plugins/file: got empty file name")
}
if v6 {
        records, err := LoadDHCPv6Records(filename)
} else {
        records, err := LoadDHCPv4Records(filename)
}
```